features/login/impl/src/main/kotlin/io/element/android/features/login/impl/accountprovider/AccountProviderProvider.kt

```diff
@@ -32,7 +31,7 @@ open class AccountProviderProvider : PreviewParameterProvider<AccountProvider> {
 }
 
 fun anAccountProvider() = AccountProvider(
-    url = LoginConstants.MATRIX_ORG_URL,
+    url = io.element.android.appconfig.AuthenticationConfig.MATRIX_ORG_URL,
```

Comment:
Maybe add an import? (probably due to refacto)

Reply:
Yes, the refactor broke the imports somehow.